definitions/daily-standup/README.md

```diff
@@ -0,0 +1,35 @@
+---
+permalink: /definitions/daily-standup/
+---
+
+# Definition of the Daily Standup
+
+> _“Dreams are, by definition, cursed with short life spans.”_
+> ~ Candice Bergen
+
+## Introduction
+
+Standup is a thing we use to inform each other on blockers or impediments. 
+We use this to share information and quickly interfere if someone needs / might 
```

Comment:
`interfere` should be `intervene` (see [this](http://english.stackexchange.com/questions/5708/whats-the-difference-between-intervene-and-interfere))